src/koans/10_runtime_polymorphism.clj

```diff
@@ -10,19 +10,19 @@
                    "!")))
 
 (defmulti diet (fn [x] (:eater x)))
-(defmethod diet :herbivore [a] __)
-(defmethod diet :carnivore [a] __)
-(defmethod diet :default [a] __)
+(defmethod diet :herbivore [a] (str (:name a) " eats veggies."))
```

Comment:
Be careful for this kind of lookup because it only works if key is a keyword in other case you need to use different notation where map is the function and key is an argument. Good [read](https://www.freecodecamp.org/news/clojure-hashmaps-explained-how-to-retrieve-values-from-and-update-hashmaps/).

Reply:
Great suggestion :)